test/functional/wallet_send.py

```diff
@@ -346,7 +346,7 @@ def run_test(self):
 
         assert_raises_rpc_error(-8, "Use fee_rate (sat/vB) instead of feeRate", w0.send, {w1.getnewaddress(): 1}, 6, "conservative", 1, {"feeRate": 0.01})
 
-        assert_raises_rpc_error(-3, "Unexpected key totalFee", w0.send, {w1.getnewaddress(): 1}, 6, "conservative", 1, {"totalFee": 0.01})
+        assert_raises_rpc_error(-3, "JSON value of type string is not of expected type bool", w0.send, {w1.getnewaddress(): 1}, 6, "conservative", 1, {"totalFee": 0.01})
```

Comment:
The new error message is simply wrong and unrelated??

Reply:
The opened issue referred to the unification of the error string. I might have understood it incorrectly, but from my interpreration the idea is to use the same error message for all JSON format errors.